testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -804,7 +804,9 @@
 end
 
 Then(/^I should only see success signs in the product list$/) do
-  raise ScriptError, 'No product synchronized' if page.has_no_xpath?('//*[contains(@class, \'fa-check-circle\')]')
+  unless page.has_xpath?('//span[contains(@class, \'text-success\')]', wait: Capybara.default_max_wait_time)
```

Comment:
Maybe introducing this, we should rename the step as:
```
When(/^I wait until I only see success signs in the product list$/) do
```

Reply:
It depends, isn't Capybara.default_max_wait_time the standard "short" wait ? In the other "I wait..." steps, we do explicitely wait for a long time.

Reply:
Ok, fine to keep it as it is if you prefer.